ui/src/components/domain/UserDomains.js

```diff
@@ -116,7 +120,24 @@ class UserDomains extends React.Component {
         });
     }
 
+    showError(errorMessage) {
+        this.setState({
+            showError: true,
+            errorMessage: errorMessage,
+        });
+    }
+
     render() {
+        if (this.state.showError) {
```

Comment:
I don't think we should show an alert error if there is an issue loading the side tab(domain list)

Reply:
We can show text in that area with the error